docs/source/en/api/pipelines/cogvideox.md

```diff
@@ -98,6 +98,12 @@ It is also worth noting that torchao quantization is fully compatible with [torc
   - all
   - __call__
 
+## CogVideoXImageToVideoPipeline
+
+[[autodoc]] CogVideoXImageToVideoPipeline
+  - all
+  - __call__
```

Comment:
If there's any restrictions on precisions (like fp16 shouldn't be used or something) that shouldn't be used we could add them in the docs.